integration_testing/features/admin/admin-sync-facility.feature

```diff
@@ -0,0 +1,24 @@
+Feature: Admin syncs facility
+  Admin needs to be able to sync their facility data to Kolibri Data Portal
+
+  Background:
+    Given I am logged in as a Facility admin
+      And my facility has been registered before
+      And I want to sync my facility data to Kolibri Data Portal
+      And I am in the Data tab in the Facility plugin
+
+  Scenario: Learn what sync does
+    When I click on *Usage and privacy* in the description of the *Sync facility data* section
+    Then I see a modal with a description of what sync does
+
+  Scenario: Successful sync
+    When I click *Sync*
+    Then I see an indeterminate loading spinner under the facility name
+      And I see *Syncing* next to the spinner
+    When syncing successfully completes
+    Then I see *Last successful sync: Just now* underneath the facility name
+
+  Scenario: Failed sync
+    When I click *Sync*
+    When the sync fails
```

Comment:
do we need a bit more instruction here on how to trigger a failure?

Reply:
We can change `KOLIBRI_DATA_PORTAL_SYNCING_BASE_URL` to a fake url so it will fail immediately. Also, makes me think that we need some availability modal in the case KDP can not be reached (but this could be for 0.13.1).

Reply:
> We can change KOLIBRI_DATA_PORTAL_SYNCING_BASE_URL to a fake url so it will fail immediately. 

Testers won't know this unless it's documented somewhere, e.g. in the Gherkin story

> but this could be for 0.13.1

Note that we cannot add user-facing strings in patch releases because we don't have translators available for those. One option would be to stub out `$trs` now, and implement the functionality later.

cc @jtamiace should we mock these up?

Reply:
added a reference to `KOLIBRI_DATA_PORTAL_SYNCING_BASE_URL` in `887c617c085604979a039f9883eeedb1fc8e21d8`